portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualDefinitions.testcase

```diff
@@ -45,8 +47,8 @@ definition {
 		ACDefinitions.goToDefinitionType(definitionType = "Individuals");
 
 		ACDefinitions.assertIndividualAttributesSource(
-			attributeList = "additionalName,address,birthDate,classPK,contactId,createDate,email,familyName,gender,givenName,jobTitle,modifiedDate,telephone,timeZoneId,userId,uuid",
-			dataSourceName = "Liferay DXP");
+			attributeList = "additionalName,address,birthDate,email,familyName,gender,givenName,jobTitle,telephone",
```

Comment:
hey @joshitagaki , is it okay to not check these attributes?

Reply:
Hey @alexcandidos, I'm not completely sure but I believe that some of those attributes I listed were never shown in the individual definitions as those are for more backend

Reply:
I see @joshitagaki ... Thanks for clarifying!